index.html

```diff
@@ -0,0 +1,106 @@
+    </header>
+    <div class="container">
+      <div class="main-video-container">
+        <iframe
+          loop
+          controls
+          class="main-video"
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/YXhIaWU3kBg"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+          allowfullscreen
+        ></iframe>
+
+        <h3 class="main-vid-title">Ist video</h3>
+      </div>
+
+      <div class="video-list-container">
+        <div class="list active">
+          <iframe
+            loop
+            controls
+            class="main-video"
+            width="360"
+            height="100"
+            src="https://www.youtube.com/embed/YXhIaWU3kBg"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <h3 class="list-title">2nd video</h3>
+        </div>
+
+        <div class="list">
+          <iframe
+            loop
+            controls
+            class="main-video"
+            width="560"
+            height="315"
+            src="https://www.youtube.com/embed/YXhIaWU3kBg"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <h3 class="list-title">3rd video</h3>
+        </div>
+
+        <div class="list">
+          <iframe
+            loop
+            controls
+            class="main-video"
+            width="560"
+            height="315"
+            src="https://www.youtube.com/embed/YXhIaWU3kBg"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <h3 class="list-title">4th video</h3>
+        </div>
+      </div>
+      <div class="partners">
+          <div class="dropdown">
+            <input type="checkbox" id="dropdown" />
+            <label for="dropdown" class="dropdown-btn" id="partners">
+              <li class="partners">Partners</li>
+              <span class="arrow right"></span>
+            </label>
+
+            <div class="dropdown-content" role="menu">
+              <div><img src="assets/images/nike.png"/><a href="#">Nike</a></div>
+              <div><img src="assets/images/addidas.png" alt="addidas
+                  image"/><a href="#">addidas</a></div>
+              <div><img src="assets/images/javascript.png" alt="gucci
+                  image"/><a href="#">Gucci</a></div>
+            </div>
+          </div>
```

Comment:
Please consider using more semantic html tags instead of div(s) like unordered lists to display the list of partners. This helps in improving SEO(search engine optimization) of your pages.